### PR TITLE
Added show password functionality

### LINKED ISSUE #158: To add - Show Password - feature in Sign Up form
**Is your feature request related to a problem? Please describe.**
While setting the password in the **Sign Up** form , user won't be able to see it, as the option is not been provided

**Describe the solution you'd like**
Adding a logic to make the password visible to user when he desires.
i.e,

From this:
athavani-from

To this:
athavani-to

**Additional context**
I am a participant of GSSOC21, and would like to solve the issue, so please assign it to me.
Thank You

### PR DESCRIPTION
**Pull request by DineshKoushik**
### Description
Added a simple eye button, which on press toggles the field in password input from password to text.

Screenshot (125)

### Related issue
closes #158

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
   "private": true,
   "proxy": "http://localhost:3000",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
```
Comment (ashishnagpal2498) [inline client/package.json:7]: No need to add this dependency. Kindly use react icons only. That dependency is already included in the project.

File: client/package.json [hunk 2]
```diff
@@ -4,6 +4,9 @@
...
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
```
Comment (ashishnagpal2498) [inline client/package.json:9]: Please remove all of these dependencies.

**Review by ashishnagpal2498 — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 2 above)
Comment (DineshKoushik) [reply client/package.json]: @ashishnagpal2498 , please check the new PR

**Comment by sanjay270899** [comment]
@DineshKoushik You were not assigned to the issue. One contributor was assigned, and he has already sent PR.

**Comment by Tejas1510** [comment]
@DineshKoushik before working on any issue just ask whether you could work on the issue as their might be someone else working on the same.
You can start working on the issue when the maintainer allots you the issue which will save your hard work and increase efficiency in maintaining the project

**Comment by Tejas1510** [comment]
Closing this PR

**Comment by DineshKoushik** [comment]
@sanjay270899 , @Tejas1510 . Firstly, the issue was open. Secondly, no one was assigned in the conversation under issue. So, how would I except to anyone work on that?

**Pull request closed without merging**